src/Task.cc

```diff
@@ -1974,7 +1974,7 @@ const string& Task::trace_dir() const {
   return trace->dir();
 }
 
-uint32_t Task::trace_time() const {
+FrameTime Task::trace_time() const {
   const TraceStream* trace = trace_stream();
   return trace ? trace->time() : 0;
 }
```

Comment:
Note: Here `trace->time()` has type `FrameTime`, so `Task::trace_time()` should also be `FrameTime`